__mocks__/axios/index.ts

```diff
@@ -0,0 +1,21 @@
+import stock from './fetch.test.stock.json';
+import fx from './fetch.test.fx.json';
+
+// NOTE: This axios mock has not been used/tested outside of FinnHub/index.test.ts
```

Comment:
The mock turned out to not be useful in tracking this bug, but it is nice to have so left it in here